### PR TITLE
Cards activity Week 4 Day 2

### PR DESCRIPTION
**Pull request by gealsanchez**
index.html updated, cards added via DOM
style.css updated, cards section styled
portfolio.js updated, cards section added via DOM
Live icon uploaded

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -534,10 +560,67 @@ footer .social {
 }
 
 .dynadiv {
-  height: 30vh;
   background-color: #fff;
 }
 
+.openover {
```
Comment (topeogunleye) [inline style.css:570]: ## I kindly suggest that you follow the Figma design by ensuring you use the same font, colours, images, text, margins). 

- [ ] Kindly ensure that your Figma Design is Responsive by ensuring the close X button shows in the Mobile design.
- [ ] Kindly maintain the same background color of white not transparent and ensure your body's background below the modal is blurred when the modal is open as seen in the Figma Design.
- [ ] For Mobile, Kindly ensure that the image is placed in the center and of the same height and width as the Figma Design. 
- [ ] For Desktop, kindly ensure that the image takes up 60% of width.
- [ ] Kindy ensure that the modal does not take up the full width and height of the page as seen in the Figma design.


| Your Design | Figma Design |
| ------------- | ------------- |
| image  | image  |
| image | image  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @gealsanchez, @Itswali,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good summary description :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -136,13 +136,18 @@ body {
...
+.closer img {
+  margin-left: 85%;
+}
+
 .social img {
   height: 20px;
 }
 
 .navigation img {
-  margin-left: auto;
```
Comment (BenMukebo) [inline style.css:144]: - [ ] Let's bring down the closed menu button and make sure it aligns on the same line as the popup menu project name (Profesional Art Printing Data)

| Your Design | Figma Design |
| ------------- | ------------- |
|  | image  |

File: style.css [hunk 3]
```diff
@@ -174,6 +179,10 @@ a {
   color: rgb(248, 246, 246);
   text-decoration: none;
   display: inline-block;
+  font-family: 'Inter', sans-serif;
```
Comment (BenMukebo) [inline style.css:186]: - [ ] Kindly fix the popup windows page on the Desktop view, have a look at the recording screen video 👇🏻Actually almost every thing need to be improved 👍🏻  

[Screencast from 10-21-2022 06:52:23 PM.webm](https://user-images.githubusercontent.com/79058364/197250030-dab123ef-afb2-4e2c-91d2-2460d299cebe.webm)

**Review by BenMukebo — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

- [OPTIONAL] This is a group project. Please let's include all the authors in the README file. This makes the README file professional and descriptive. 


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (gealsanchez) [reply style.css:186]: Hello @BenMukebo I test the changes in two different machines and I did not see the errors you show in the video, how can I check to be sure that my changes are ok? thank you

**Review by AnselemOdims — APPROVED** [review]
### Hi  @gealsanchez 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**